hazelcast/src/test/java/com/hazelcast/jet/core/JobSerializerTest.java

```diff
@@ -71,22 +69,15 @@
     public static void beforeClass() {
         Config config = smallInstanceConfig();
         config.addCacheConfig(new CacheSimpleConfig().setName(SOURCE_CACHE_NAME))
-              .addCacheConfig(new CacheSimpleConfig().setName(SINK_CACHE_NAME));
+                .addCacheConfig(new CacheSimpleConfig().setName(SINK_CACHE_NAME));
 
         ClientConfig clientConfig = new ClientConfig();
         clientConfig.getSerializationConfig()
-                    .addSerializerConfig(new SerializerConfig().setTypeClass(Value.class).setClass(ValueSerializer.class));
+                .addSerializerConfig(new SerializerConfig().setTypeClass(Value.class).setClass(ValueSerializer.class));
 
         initializeWithClient(1, config, clientConfig);
     }
 
-    @Test
-    public void when_serializerIsNotRegistered_then_mapThrowsException() {
```

Comment:
Maybe instead of removing the test, we can put `new Object()` as we did above?

Reply:
I have reverted the tests. Object is serializable as well right now since we are not checking `java*` class name anymore. 

The tests are passing right now because compact is disabled by default. We will came accross these tests failures when we enabled it by default. Postponing this problem to the future self.